bundle/manifests/configmap.yaml

```diff
@@ -0,0 +1,676 @@
+apiVersion: v1
```

Comment:
Kind 'Configmap' is supported in bundle format. OLM operator with version 0.15.0+ is required to support 'Configmap'.

Reply:
> Thanks for checking this. Did you check that OCP 4.6 and above all have a compliant OLM?

Yes, I had checked OCP 4.6(with OLM 0.16.1) and 4.7(with OLM 0.17.0). I didn't check OCP 4.8 as I don't have OCP 4.8 environment, but I think the OLM version won't be below 0.15.0 on OCP 4.8.